src/ewah.zig

```diff
@@ -2,7 +2,9 @@ const std = @import("std");
 const assert = std.debug.assert;
 const math = std.math;
 const mem = std.mem;
-const div_ceil = @import("util.zig").div_ceil;
```

Comment:
Could util.div_ceil be replaced with `std.math.divCeil(...) catch unreachable`?

Reply:
See discussion here: https://github.com/tigerbeetledb/tigerbeetle/pull/62#discussion_r791035331

Reply:
I see :)